MembraneRTC/Sources/MembraneRTC/Events/Event.swift

```diff
@@ -295,9 +295,14 @@ struct OfferDataEvent: ReceivableEvent, Codable {
     }
 
     struct Data: Codable {
-        let iceTransportPolicy: String
         let integratedTurnServers: [TurnServer]
         let tracksTypes: [String: Int]
+        
+        init(from decoder: Decoder) throws {
+            let container = try decoder.container(keyedBy: CodingKeys.self)
+            self.integratedTurnServers = try container.decode([TurnServer].self, forKey: .integratedTurnServers)
```

Comment:
```suggestion
```

Reply:
all this `init(from: Decoder)` is not needed here anymore as we don't do any custom decoding anymore. Removed @graszka22